### DISCUSSION TITLE
NWM v2.1 streamflow at all nhd flowlines

### DISCUSSION THREAD

**Discussion by sfoks** [hytest-org/hytest#459] [Data Storage and Access]
Hi, @csimeone-usgs and myself are interested in whether anyone has accessed and created an ARCO dataset of the [CHRTOUT output files](https://ral.ucar.edu/sites/default/files/public/WRFHydroV5_OutputVariableMatrix_V5.pdf) from NWM v2.1 ([files here](https://noaa-nwm-retrospective-2-1-pds.s3.amazonaws.com/index.html#model_output/))? We are interested in streamflow.  @amsnyder @rsignell do you know if this exists somewhere?

**Comment by rsignell** [comment]
@sfoks NWM3.0 is on AWS in both NetCDF and zarr formats. For example: s3://noaa-nwm-retrospective-3-0-pds/CONUS/zarr/chrtout.zarr
(And sorry for the delay! I made some inquiries, eventually got the answers and then forgot to pass them along!) 😳
